matchzoo/preprocessors/units/vocabulary.py

```diff
@@ -27,26 +27,14 @@ class Vocabulary(StatefulUnit):
         >>> c_index = term_index['C']
         >>> vocab.transform(['C', 'A', 'C']) == [c_index, a_index, c_index]
         True
-        >>> vocab.transform(['C', 'A', 'OOV']) == [c_index, a_index, 1]
+        >>> vocab.transform(['C', 'A', '<OOV>']) == [c_index, a_index, 1]
         True
         >>> indices = vocab.transform(list('ABCDDZZZ'))
         >>> ' '.join(vocab.state['index_term'][i] for i in indices)
-        'A B C D D OOV OOV OOV'
+        'A B C D D <OOV> <OOV> <OOV>'
 
     """
 
```

Comment:
I suggest adding `OOV = "<OOV>"` and `PAD = "<PAD>"` here, and use it like `self.OOV` or, from the user's perspective, `Vocabulary.OOV`.

Reply:
done~